src/pod.rs

```diff
@@ -1,9 +1,11 @@
 //! Plain data object safe transmute
 //!
-//! Functions in this module are guarded from out-of-bounds memory access as well as from unaligned access,
-//! raising errors on both cases. Moreover, only a safe subset of types (which implement
-//! [`PodTransmutable`](trait.PodTransmutable.html)) can be used as the transmute target, thus ensuring
-//! full safety.
+//! Functions in this module are guarded from out-of-bounds memory access and from unsafe transmute
+//! target types with the use of the [`PodTransmutable`](trait.PodTransmutable.html)) trait.
+//! 
+//! However, they are not entirely safe because the value may be properly aligned for reading and
+//! writing a value of that type. The effects of this range from less performance (e.g. x86) to
+//! trapping or address flooring (e.g. ARM CPUs), but this is undefined behavior nonetheless.
```

Comment:
Missing a negative in the first sentence, x86/ARM either both need to be specified as "CPUs" or need both be unburdened.

The first sentence sounds like it would make much more sense in some other language, too – too many "value"s make it confusing.